OneDrive-Cloud-Player/OneDrive-Cloud-Player.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>OneDrive-Cloud-Player</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
+	<TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
```

Comment:
I think your Visual Studio might be replacing spaces with tabs. At least there were spaces before, and there are tabs now. So maybe revert that.

Reply:
I will take a look

Reply:
Oh i was using Visual Studio Code for this change.